Index: jessie-stage2/rootfs/etc/dhcpcd.conf
===================================================================
--- jessie-stage2.orig/rootfs/etc/dhcpcd.conf
+++ jessie-stage2/rootfs/etc/dhcpcd.conf
@@ -57,3 +57,7 @@
 # fallback to static profile on eth0
 #interface eth0
 #fallback static_eth0
+
+# Disable link-local addressing completely
+# This seems to cause issues when DHCP times out (?) on the FRC radio
+noipv4ll
